server/hl7/hl7.controller.js

```diff
@@ -70,6 +70,9 @@ function parseFile(req, res, next) {
   }).catch(err =>
     next(new APIError(err, httpStatus.BAD_REQUEST))
   ).then((data) => {
+    if (!data.startsWith('MSH|')) {
+      throw new Error('Please upload a valid HL7 file');
```

Comment:
Throwing an error here like this will cause the sever to return a `500`. We can instead do this:
`next(new APIError('Please upload a valid HL7 file', httpStatus.BAD_REQUEST));` - this will return a `400 Bad Request`